### PR TITLE
[7.0] new module: sql_export_email

### PR DESCRIPTION
**Pull request by laetitia-gangloff**
This PR is based on
- [x] #115 
- [x] #175

It is to send by email the result of an sql export

### CODE REVIEW

File: sql_export_email/ir_actions_view.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                                                          ('state','!=','dummy'),
+                                                          ('state','!=','sms'),
+                                                          ('state','!=','code'),
+                                                          ('state','!=','loop'),
+                                                          ('state','!=','trigger'),
+                                                          ('state','!=','object_copy'),
+                                                          ('state','!=','client_action'),
+                                                          ('state','!=','email'),
+                                                          ('state','!=','sms'),
+                                                          ('state','!=','other')]}</attribute>
```
Comment (JonathanNEMRY) [inline sql_export_email/ir_actions_view.xml:27]: I think you can use 'state', 'not in', ['sql_export_email',...]

File: sql_export_email/ir_actions.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    _inherit = "ir.actions.server"
+
+    def __init__(self, pool, cr):
+        super(actions_server, self).__init__(pool, cr)
+        # add state 'sql_export_email'
+        new_state = ('sql_export_email', _('Export SQL data by email'))
+        if new_state not in self._columns['state'].selection:
+            self._columns['state'].selection.append(new_state)
+        return
+        return
```
Comment (JonathanNEMRY) [inline sql_export_email/ir_actions.py]: I think you can remove one of them :)

File: sql_export_email/__init__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from . import ir_actions
```
Comment (JonathanNEMRY) [inline sql_export_email/__init__.py:3]: Header is not required into the **init**.py

File: sql_export_email/ir_actions_view.xml (hunk 1 above)
Comment (laetitia-gangloff) [inline sql_export_email/ir_actions_view.xml:27]: Yes, but I prefer to keep the same syntax as original view :
https://github.com/odoo/odoo/blob/7.0/openerp/addons/base/ir/ir_actions.xml#L358

I hope to find a better way to write this (need to explore fields_view_get)

**Comment by lmignon** [comment]
@laetitia-gangloff can you rebase?

File: sql_export_email/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    "name": "SQL export email addon",
+    "version": "0.1",
+    "author": "ACSONE SA/NV",
```
Comment (gurneyalex) [inline sql_export_email/__openerp__.py]: Can you add "Odoo Community Association (OCA)" to the authors list? (and rebase)

**Comment by gurneyalex** [comment]
@laetitia-gangloff can you rebase?

**Comment by laetitia-gangloff** [comment]
@gurneyalex I rebased, thank you !

**Comment by gurneyalex** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:

**Comment by laetitia-gangloff** [comment]
Is it not mergeable ? 2 thumb up and more than 5 days ?

**Comment by moylop260** [comment]
:+1:

**Pull request merged**